processor/groupbytraceprocessor/processor.go

```diff
@@ -128,50 +123,57 @@ func (sp *groupByTraceProcessor) processResourceSpans(rs pdata.ResourceSpans) er
 	}
 
 	for _, batch := range splitByTrace(rs) {
-		traceID := batch.traceID
-		if sp.ringBuffer.contains(traceID) {
-			// it exists in memory already, just append the spans to the trace in the storage
-			if err := sp.addSpans(traceID, batch.rs); err != nil {
-				return fmt.Errorf("couldn't add spans to existing trace: %w", err)
-			}
-
-			// we are done with this trace, move on
-			return nil
+		if err := sp.processBatch(batch); err != nil {
+			sp.logger.Info("failed to process batch", zap.Error(err))
```

Comment:
Might be useful to add traceID associated with the batch into message

Reply:
Great idea. Added.